presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -989,6 +990,7 @@ private void printTableScanInfo(TableScanNode node, int indent)
                     predicate.getDomains().get()
                             .entrySet().stream()
```

Comment:
We can simplify this by changing to `keySet()`, since we don't ever use the value.